bin/configurer/setup-local-idam.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eu
+
+dir=$(dirname ${0})
+
+# Defaults are ok and create require ccd_gateway service in idam
+#${dir}/utils/create-ccd-service.sh
+
+# citizen / solicitor created by default
+${dir}/utils/idam-add-role.sh "ccd-import"
```

Comment:
There is `add-roles.sh` script already that is supposed to register roles that service needs with IDAM and CCD. I'd recommend moving these few lines there.

I think having one 'go-to' file in case new role is added is better than having two files.

Reply:
Ok, I separated the IDAM stuff here, but I think you are right I will refactor a bit and put all roles in single place.